src/pages/UploadPage.tsx

```diff
@@ -40,15 +74,15 @@ export default function UploadPage() {
           label="Cover Letter"
           defaultValue="Insert File here"
           size="small"
-          sx={{ width: '300px', marginRight: 2 }}
+          sx={{ width: '300px', marginRight: 2}}
           value={cover_letter?.name}
         />
         <Button
           size="small"
           variant="contained"
           component="label"
           color="secondary"
-          sx={{ width: '100px' }}
+          sx={{ width: '100px',  height: '40px' }}
```

Comment:
```suggestion
          sx={{ width: '100px',  height: '40px' }}
          disableElevation
```
Disable elevation to keep a consistent button look throughout the application.